lib/adroll/api/advertisable.rb

```diff
@@ -15,7 +15,7 @@ def create(params)
       end
 
       def edit(params)
-        call_api(:put, __method__, sanitize_params(params))
+        call_api(:post, __method__, sanitize_params(params))
```

Comment:
Do me a favor... tell Devin to fix this. It hurts me every time I see it. Updates are supposed to PUTs, not POSTs.